lib/authentication/SecureStorage/credentialManager.js

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
I think it's good idea change the name to jsonCredentialManager.js and here import localStorage.js logic

Reply:
Done, and merged.